### DISCUSSION TITLE
Captcha system / Stones hit count

### DISCUSSION THREAD

**Discussion by popalexandrudan** [sunflower-land/sunflower-land#2519] [Ideas]
Is there a reason why the captcha appears so often ? I'm 100% confident, that its only annoying for the players, and not stopping any more bots than it would stop, if it happened way less often. 

Also, is there a reason why we need to click stones 3 times? Its pretty annoying on mobile. I think hitting it 2 times only, would be very nice.

**Comment by offerT** [comment]
I also believe that the Captcha system needs to be reduced

**Comment by future2back** [comment]
Captcha system is related to mutant crops, if reduce it, mutant crops will be reduced. Applied to seeds and gold drops if you have the skill.

We have to stop with these expansions, this generation of ever-increasing resources... more and more clicks in game and little entertainment.

**Comment by popalexandrudan** [comment]
can this get a little of attention please?
doing 1000 SFL quest and seeing captcha so often drives me crazy.

at the moment, you are not keeping bots away, you are making old players burn out.

also, can you guys add what you did for trees to stones/iron/gold as well? 
cutting trees now feels very smooth
